staff/luis-garrido/e-commerce-protos/scripts/cart.test.js

```diff
@@ -0,0 +1,159 @@
+suite("cart", function () {
```

Comment:
You should implement also the negative tests of the functions that throw errors 🤓

Reply:
I should and I'll do!